main.py

```diff
@@ -2,6 +2,9 @@
 from supporting_components.graph_io import *
```

Comment:
Misschien dat we iets met deze filenaam moeten. Het is eigenlijk geen main meer. Misschien kun je van deze file dan meer iets maken van dat je gewoon de csvwriter test met de color refinement zoals dat nu gebeurt. En het dan ook in de testmap zetten.

Reply:
Ja hier ben ik het heel erg mee eens. Eigenlijk wat onze main eerst was, zijn nu de tests in de tests map en main gebruiken we eigenlijk niet meer.

Reply:
ik heb hem verplaatst naar de tests map